Library/Homebrew/software_spec.rb

```diff
@@ -541,11 +541,11 @@ def ==(other)
   end
   alias eql? ==
 
-  sig { params(tag: Utils::Bottles::Tag).returns(T.any(Symbol, String)) }
+  sig { params(tag: Utils::Bottles::Tag).returns(String) }
   def tag_to_cellar(tag = Utils::Bottles.tag)
     spec = collector.specification_for(tag)
     if spec.present?
-      spec.cellar
+      spec.cellar.to_s
     else
       tag.default_cellar
     end
```

Comment:
This doesn't seem correct. Cellar can be both a symbol and a string, and it looks like this is breaking CI as a result.

Reply:
Ah, ty! Fixed in https://github.com/Homebrew/brew/pull/16183/commits/3c0506e2760ef82f769871eb46eb01db8fa72fde